apps/cnspec/cmd/bundle.go

```diff
@@ -165,16 +167,19 @@ var policyPublishCmd = &cobra.Command{
 			log.Fatal().Err(err).Msg("could not find bundle files")
 		}
 
-		result, err := bundle.Lint(files...)
-		if err != nil {
-			log.Fatal().Err(err).Msg("could not lint bundle files")
-		}
+		noLint := viper.GetBool("no-lint")
+		if !noLint {
+			result, err := bundle.Lint(files...)
+			if err != nil {
+				log.Fatal().Err(err).Msg("could not lint bundle files")
+			}
 
-		// render cli output
-		os.Stdout.Write(result.ToCli())
+			// render cli output
+			os.Stdout.Write(result.ToCli())
 
-		if result.HasError() {
-			log.Fatal().Msg("invalid policy bundle")
+			if result.HasError() {
+				log.Fatal().Msg("invalid policy bundle")
+			}
 		}
 		log.Info().Msg("valid policy bundle")
```

Comment:
At this point we are not sure it is valid. Compile could fail server-side. I suggest we only show that when the linting was completed successfully.

Reply:
I'm not sure I understand. Is the ask to always upload the bundle, regardless of what the lint says?

Reply:
If you call --no-lint we essentially not verify the bundle on client side, so we cannot say at this point that the bundle is actually valid.